.env

```diff
@@ -1,2 +1,3 @@
 EXPO_PUBLIC_SUPABASE_URL="https://topix.site"
 EXPO_PUBLIC_SUPABASE_ANON_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.ewogICJyb2xlIjogImFub24iLAogICJpc3MiOiAic3VwYWJhc2UiLAogICJpYXQiOiAxNjk5OTE2NDAwLAogICJleHAiOiAxODU3NzY5MjAwCn0.UUqrgem6oFsSJWV4fhPx38TwQJd00bw6GZ781pCJQs4"
+EXPO_PUBLIC_TOPIX_API_KEY='9a033a0b-a8d0-463e-adc9-eb556da6c5f8'
```

Comment:
If this is removed and regenerated (preferably in that order), the pr can be merged